src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -938,7 +938,7 @@ void ClassLoader::load_java_library() {
 }
 
 void ClassLoader::release_load_zip_library() {
-  MutexLocker locker(Zip_lock, Monitor::_no_safepoint_check_flag);
+  ConditionalMutexLocker locker(Zip_lock, Zip_lock != nullptr, Monitor::_no_safepoint_check_flag);
```

Comment:
Why is this one now `ConditionalMutexLocker`?

Reply:
This one we know about: https://bugs.openjdk.org/browse/JDK-8313210